msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Use Microsoft Graph to enable proactive bot installation and messaging in Teams
+description: Describes proactive messaging in Teams and how to implement.
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: Overview
+keywords: teams proactive messaging chat installation Graph
+---
+# Enable proactive bot installation and proactive messaging in Teams with Microsoft Graph (Public Preview)
+
+>[!IMPORTANT]
+> Microsoft Graph public previews are available for early-access and feedback. Although this release has undergone extensive testing, it is not intended for use in production.
+
+## Proactive messaging in Teams
+
+Proactive messages are initiated by bots to start conversations with a users. They serve many purposes including sending welcome messages, conducting surveys or polls, and broadcasting organization-wide notifications.  Proactive messages in Teams can be delivered as either **ad-hoc** or **dialog-based** conversations:
+
+|Message Type | Description |
+|----------------|-------------- |
+|Ad-hoc proactive message| The bot interjects a message without interrupting the conversation flow.|
+|Dialog-based proactive message | The bot creates a new dialog thread, takes control of a conversation, delivers the proactive message, closes, and returns control to the previous dialog.|
+
+*See*, [Send proactive notifications to users SDK v4](/azure/bot-service/bot-builder-howto-proactive-message?view=azure-bot-service-4.0&tabs=csharp)
+
+## Proactive app installation in Teams
+
+Before your bot can proactively message a user, it needs to be installed either as a personal app, or in a team where the user is a member. At times,  you may need to proactively message users that have _not_ installed or previously interacted with your app. For instance, the need to message vital information to everyone in your organization. For such scenarios, you can use the Microsoft Graph API to proactively install your bot for your users.
+
+## Permissions
+
+Microsoft Graph [teamsAppInstallation resource type](/graph/api/resources/teamsappinstallation?view=graph-rest-1.0) permissions allow you to manage your app's installation lifecycle for all user (personal ) or team (group) scopes within the Microsoft Teams platform:
```

Comment:
I would avoid using the term 'group' because it will conflate with a another concent 'group chats'.